colour/models/cie_xyy.py

```diff
@@ -134,6 +138,112 @@ def xyY_to_XYZ(xyY):
     return XYZ
 
 
+def xy_to_xyY(xy):
+    """
+    Converts from *xy* chromaticity coordinates to *CIE xyY* colourspace.
+
+    `xy` argument with last dimension being equal to 3 will be assumed to be a
```

Comment:
Now that I was adding the argument, I found it a bit weird actually to have that option, in cases we need Luminance to be passed, shouldn't it be through an *xyY* array?